src/plugins/elasticsearch/index.js

```diff
@@ -0,0 +1,149 @@
+const elasticsearch = require('elasticsearch');
+const accessLogsIndexConfig = require('./access-logs_index.json');
```

Comment:
Any ideas for a better name for "access-logs_index.json"?

Reply:
So, if the config is for the access-watch-logs indexes, we can name the variable `accessWatchLogsIndexConfig`, a bit verbose but logic.

And then the file, `access-watch-logs-index-config.json`, same, verbose but clear.

We can also consider extracting the mapping out of the config and have the file be `access-watch-logs-index-mapping.json` but then, if we need to pass more than the mapping, we would need something else.